bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1407,7 +1408,7 @@ namespace bftEngine
 
 			if (askForStateTransfer)
 			{
-				LOG_INFO_F(GL, "call to startCollectingState()");
```

Comment:
This is probably useful for debugging during testing, but we can always turn on debug mode. I don't think it's that frequent, so it may be beneficial to leave it at INFO, in case we ship a bug where state transfer isn't turning on for some reason.

Reply:
I will return in to `INFO`.